library/core/src/panic/panic_info.rs

```diff
@@ -128,6 +70,24 @@ impl<'a> PanicInfo<'a> {
         Some(&self.location)
     }
 
+    /// Returns the payload associated with the panic.
+    ///
+    /// On `core::panic::PanicInfo`, this method never returns anything useful.
+    /// It only exists because of compatibility with [`std::panic::PanicHookInfo`],
+    /// which used to be the same type.
+    ///
+    /// See [`std::panic::PanicHookInfo::payload`].
+    ///
+    /// [`std::panic::PanicHookInfo`]: ../../std/panic/struct.PanicHookInfo.html
+    /// [`std::panic::PanicHookInfo::payload`]: ../../std/panic/struct.PanicHookInfo.html#method.payload
+    #[deprecated(since = "1.77.0", note = "this never returns anything useful")]
```

Comment:
The "since" here seems wrong.
There's some sort of magic string that can be put here to become the current version on the next bootstrap bump, but I can never remember the exact string...

Reply:
Looks like that magic string doesn't work in `#[deprecated]`, only in `#[stable]`. I'll update it to the correct version (1.81.0).